server/controllers/Users.js

```diff
@@ -86,6 +86,24 @@ class Users {
       return serverError(res);
     }
   }
+
+  /**
+   * @name changePassword
+   * @async
+   * @static
+   * @memberof Users
+   * @param {Object} req express request object
+   * @param {Object} res express response object
+   * @returns {JSON} JSON object with details of new user
+   */
+  static async changePassword(req, res) {
+    const { password } = req.body;
+    const { id } = req.user.dataValues;
+    await User.update({ password }, { where: { id } });
```

Comment:
I believe you should hash this password before saving it to the database. This way, when the user tries to login, the `bcrypt.compare()` function will be able to do a successful comparison.

Reply:
yes I have hashed it thanks